com.her01n.BatteryInfo.json

```diff
@@ -80,6 +80,20 @@
           "sha256": "7da7ac358f5967c60bbbc72ad106b496c3e4b9d08b8a6d7409da6b3551371531"
         }
       ]
+    },
+    {
+      "name": "cleanup",
+      "buildsystem": "simple",
+      "build-commands": [
+        "find /app/lib -name '*.a' -delete",
+        "find /app/lib -name '*.la' -delete",
+        "rm -rf /app/include",
+        "rm -rf /app/lib/pkgconfig",
+        "rm -rf /app/share/aclocal",
+        "rm -rf /app/share/doc",
+        "rm -rf /app/share/info",
+        "rm -rf /app/share/man"
+      ]
```

Comment:
Not like this, https://github.com/flathub/com.github.calo001.fondo/blob/f937df8640aa2a8058743c89986f278ab3acd23c/com.github.calo001.fondo.json#L24-L35